lib/dark_worlds_server/bot/bot_client.ex

```diff
@@ -0,0 +1,65 @@
+defmodule DarkWorldsServer.Bot.BotClient do
+  alias DarkWorldsServer.Communication
+  alias DarkWorldsServer.Communication.Proto.GameEvent
+  alias DarkWorldsServer.RunnerSupervisor.BotPlayer
+  use WebSockex, restart: :temporary
+  require Logger
+
+  def start_link(%{game_id: game_id, config: config}) do
+    client_id = UUID.uuid4()
+    ws_url = Application.fetch_env!(:dark_worlds_server, DarkWorldsServer.Bot) |> Keyword.get(:game_server_url)
+    WebSockex.start_link("#{ws_url}/play/#{game_id}/#{client_id}/2", __MODULE__, %{config: config}, [])
+  end
+
+  @impl true
+  def handle_connect(_conn, state) do
+    Logger.info("Connected #{inspect(self())}")
```

Comment:
Should we keep this?

Reply:
Like the others, don't see any harm. Is not he most useful log, but is not useless and we should start thinking about logs in the app

Reply:
Sounds good!